### ISSUE TITLE
Image path not found in local dev-server with abell serve

### ISSUE THREAD

**Issue by saurabhdaware** [abelljs/abell#117]
**Describe the bug**
In local dev server, if you add image to blog from `assets/image.jpg` path. The image does not load when we don't have `/` in the end of the URL.

**To Reproduce**

**Expected behavior**

**Screenshots**

**Specs:**

- Abell Version:
- Node Version:
- Operating System:

**Additional context**

**Comment by saideepesh000** [comment]
Hey @saurabhdaware, I want to work on this.

**Comment by saurabhdaware** [comment]
awesome! Go ahead @saideepesh000. 

So one solution I can think of is to have `/` appended by default in dev-server when the path is something like `localhost:5000/hello`. So the new path should become `localhost:5000/hello/` instead.

**Comment by saideepesh000** [comment]
It is working fine without `/` at the end

![gif](https://user-images.githubusercontent.com/43727167/110906967-bef0fd00-8332-11eb-9adc-2ba89ab9e099.gif)

**Comment by saurabhdaware** [comment]
no I meant when you add this image inside blog with `![]()` and then open the blog without `/` (`localhost:5000/my-blog`)

**Comment by saideepesh000** [comment]
> 
> 
> awesome! Go ahead @saideepesh000.
> 
> So one solution I can think of is to have `/` appended by default in dev-server when the path is something like `localhost:5000/hello`. So the new path should become `localhost:5000/hello/` instead.

Hey @saurabhdaware, what I observed is by default it is going to append `/` at the end of url like `http://localhost:5000/getting-started-with-abell/` and yes img is rendered correctly, but as u said if we don't have a `/`, img isn't rendered.

![today](https://user-images.githubusercontent.com/43727167/111077219-bb699b80-8515-11eb-8927-e7bf0663b88e.gif)

**Comment by saurabhdaware** [comment]
oh I meant when we directly hit the http://localhost:5000/hello url we need to append `/` to URL from abell's dev server

**Comment by saideepesh000** [comment]
Hey @saurabhdaware, just a setup related question, so I want to see the changes made in the [src ](https://github.com/abelljs/abell/tree/main/src)folder of the repo, for ex: I want to add the console.log(url)( See [here](https://github.com/abelljs/abell/blob/f3875c120112743be8e7599a32eeba4fb30f0dbf/src/abell-dev-server/http-server.js#L72) ) and want to see the changes in the console, so.., how to do so(how to run the local version of abell).

**Comment by saurabhdaware** [comment]
There is info on how to run local abell version- https://github.com/abelljs/abell/blob/main/CONTRIBUTING.md#local-setup

Once you get local abell version installed, you can just run `abell build` or `abell serve` in example project to see the changes

**Comment by saideepesh000** [comment]
> 
> 
> There is info on how to run local abell version- https://github.com/abelljs/abell/blob/main/CONTRIBUTING.md#local-setup
> 
> Once you get local abell version installed, you can just run `abell build` or `abell serve` in example project to see the changes

Ya, I could able to run the project and all, but I couldn't able to see the changes made in `src` folder of the repo, as I said console.log(anything) is giving anything in the terminal.

**Comment by saurabhdaware** [comment]
where exactly you're doing the console.log? can you try doing it somewhere in `src/commands/build.js` and run `abell build`

**Comment by saurabhdaware** [comment]
Since it's a server, that should be called on `abell serve` when you visit the page

**Comment by saideepesh000** [comment]
> 
> 
> Since it's a server, that should be called on `abell serve` when you visit the page

Ohhhhhh, Now I got this, whenever I made a change in those files, I need to restart the server, lol, dumbness of me 💯

Now, u can delete these comments.

**Comment by saurabhdaware** [comment]
yes unfortunately the server is for user-facing files and not the source code 🤕
